cli/compile/compile.go

```diff
@@ -100,6 +104,12 @@ func NewCommand() *cobra.Command {
 		tr("List of custom build properties separated by commas. Or can be used multiple times for multiple properties."))
 	compileCommand.Flags().StringArrayVar(&buildProperties, "build-property", []string{},
 		tr("Override a build property with a custom value. Can be used multiple times for multiple properties."))
+	compileCommand.Flags().StringVar(&keysKeychain, "keys-keychain", "",
```

Comment:
At first, I asked the same question, but having these three flags instead of 2 results in a more future-proof implementation:
In the future, `keys-keychain` could point to a real keychain instead of a simple directory and the other two flags could indicate the actual name of the keys inside the keychain.

Reply:
Make sense. Thanks for the clarification.